cmd/guacone/cmd/files.go

```diff
@@ -63,13 +75,42 @@ var exampleCmd = &cobra.Command{
 			viper.GetString("gdbpass"),
 			viper.GetString("gdbaddr"),
 			viper.GetString("realm"),
+			viper.GetString("keyPath"),
+			viper.GetString("keyID"),
```

Comment:
can we maybe add a prefix for some of these arguments, maybe `verifier-keyPath` and `verifier-keyID`